packages/table/src/Cell/Cell.styles.ts

```diff
@@ -94,7 +99,7 @@ export const cellContentTransitionStyles: Record<TransitionStatus, string> = {
   entered: css`
     opacity: 1;
     min-height: ${standardCellHeight}px;
-    max-height: ${standardCellHeight}px;
+    max-height: 40vh;
```

Comment:
changing this degrades the animation significantly. See the 10% speed clips attached 

https://github.com/mongodb/leafygreen-ui/assets/2414030/c9aea06b-d24a-4a2a-b0c6-adf29fd38212


https://github.com/mongodb/leafygreen-ui/assets/2414030/d36e0fca-17a2-4d8f-a045-e50bf2d73939

Reply:
Resolved with a little hack with the max-heights right before animations begin - I tested it on 10% speed too and it looks good to me. Let me know if you see the same with the updates

Reply:
I'm still seeing a kind-of 2-stage effect going on

https://github.com/mongodb/leafygreen-ui/assets/2414030/c9a4a9e0-2b4c-442d-9fe1-a8c54e451fb3